coralproject/talk
# [CORL-2934]: More DSA polish

## What does this PR do?

These changes make a number of design UI and copy updates related to DSA feature QA.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

* Check out the updated copy on the report form
* The confirmation screen is updated after you submit a report (test with both successful submissions and a submission that errors)
* Go into user history drawer for a commenter that has had a comment rejected for illegal content. See that there is a marker for `Illegal content` next to that comment in the `Rejected` tab
* Add additional comments to a DSA report in the report form and see the updated/streamlined flow
* In the admin, on the single DSA Report page, click `Decision` button to open the make a decision modal and see the updated styles/functionality there
* There is now a cursor pointer shown when hovering over the Reports table in the admin
* Clicking on the `Comments` tab from the report form page (and permalink page) now goes back to all comments
* If you add a really long text input to the `law broken` input, it will be truncated when shown in the Reports table
* When clicking `Report` and seeing the report illegal content link in the report form for a comment, the link copy is updated and an icon added
* on page load, if the report form isn't in view, it is now scrolled down to

## Where any tests migrated to React Testing Library?



## How do we deploy this PR?
